ci-operator/step-registry/ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/etcd/on-ramfs/ipi-etcd-on-ramfs-ref.yaml",
```

Comment:
nitpick: why not `ipi/conf/etcd-on-ramfs`?

Reply:
you suggested ipi-etcd-on-ramfs as the step so I thought that is what you meant

Reply:
sorry, my fault. I've realized other platforms could use that, so its worth moving to shared config dir